src/components/OrderPage/OrderGroup.tsx

```diff
@@ -0,0 +1,110 @@
+import styled from 'styled-components';
+import { OrderItem } from './OrderItem';
+import { OrderProductInfo } from '../../recoil/atoms/orderAtom';
+import { useNavigate } from 'react-router-dom';
+import { Fragment } from 'react';
+
+interface OrderGroupProps {
+  orders: OrderProductInfo[];
+  orderId: number;
+  totalProductPrice?: number;
+  totalOrderLength?: number;
+  isDetailPage?: boolean;
+}
+
+export const OrderGroup = ({
+  orders,
+  orderId,
+  isDetailPage,
+  totalProductPrice,
+  totalOrderLength,
+}: OrderGroupProps) => {
+  const navigate = useNavigate();
+
+  return (
+    <Style.Container>
+      <Style.Header>
+        <Style.OrderNumber $isDetailPage={isDetailPage ?? false}>
+          주문 번호: {orderId}
+        </Style.OrderNumber>
+        <Style.ViewDetailButton
+          onClick={() => {
+            navigate('/orderDetail', { state: { orderId } });
+          }}
+          $isDetailPage={isDetailPage ?? false}
+        >
+          상세보기
+        </Style.ViewDetailButton>
+      </Style.Header>
+      {totalProductPrice ? (
+        <Fragment>
```

Comment:
```suggestion
        <>
```

key prop 을 안쓸거면 굳이 Fragment라고 안써도 될거 같아요
(이렇게 쓰면 매우 old한 코드로 보인답니다. 이유는 .. 안알랴줌)

Reply:
ㅠㅠ 그런지도 모르고 좋다고 Fragment를 남발하고 있었네요. 이유는 모르지만 올드한 개발자라는 소리는 듣고 싶지 않아서 당장 다 지워버렸습니다👍

수정한 커밋: b9d0144335634838a1363d270e278fb4e38503dc f98bcbe36347230aa526f0502b2e22c6e032a8d7

Reply:
이유는, 

`<></>` 가 `<Fragment></Fragment>` 보다 나중에 나왔기 때문에

`<Fragment></Fragment>` 를 쓰면 예전 리액트 지식으로 개발하는 사람처럼 보이기 때문입니다 😄